### DISCUSSION TITLE
define personalized shortcuts with QR-Codes

### DISCUSSION THREAD

**Discussion by mahhyyar** [gopro/labs#169] [Feature Requests]
Hi everyone,

I'm using the latest GoPro Labs firmware on my H9 and when using QRControl app, I cannot define on-screen shortcuts. It would be super useful to be able to add shortcuts when defining a profile on QRControl app.

Cheers

**Comment by dnewman-gpsw** [comment]
Not currently supported on any camera model as there are no QR codes for this action. It might be doable in future firmware. It has been added to the backlog.
